src/sentry/api/endpoints/release_files.py

```diff
@@ -148,6 +149,9 @@ def post(self, request, project, version):
         if not full_name:
             return Response({'detail': 'File name must be specified'}, status=400)
         name = full_name.rsplit('/', 1)[-1]
+        pattern = re.compile(r"(\n|\t|\r|\f|\v|\\n|\\t|\\r|\\f|\\v)")
```

Comment:
You can shorten this to `[\n\t\r\f\v\\]`. This rejects all those whitespaces plus backslashes which also make no sense there.